pkg/asset/manifests/gcp/cluster.go

```diff
@@ -182,3 +207,29 @@ func findFailureDomains(installConfig *installconfig.InstallConfig) []string {
 
 	return zones.UnsortedList()
 }
+
+// getSubnet will find a subnet in a project by the name. The matching subnet structure will be returned if
+// one is found.
+func getSubnet(project, region, subnetName string) (*compute.Subnetwork, error) {
+	ctx := context.Background()
```

Comment:
Nit: I think it'd be better to receive a context as argument and then pass `context.TODO()` until `GenerateClusterAssets` itself receives a context:
```suggestion
func getSubnet(ctx context.Context, project, region, subnetName string) (*compute.Subnetwork, error) {
```